src/stories/Library/material-grid/material-grid.scss

```diff
@@ -0,0 +1,30 @@
+.material-grid {
+  @include layout-container($layout__max-width--large, 0);
+}
+
+.material-grid__title {
+  @include typography($typo__h2);
+  @include layout-container();
+
+  @extend %title-underline--narrow;
+  margin-bottom: calc($s-lg * 2);
+}
+.material-grid__items {
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+}
+
+.material-grid__items .recommended-material {
+  max-width: unset;
+  border: 1px solid $color__global-tertiary-1;
+}
```

Comment:
Please look into how we can avoid referencing another component.

According to our BEM guidelines one block should not modify the style of another block. Either we should look into if we can add modifiers to `.recommended-material` to make it behave like we want or we should create a `.material-grid__item` element within this block, style it appropriately and use this class on the `.recommended-material` elements.